.taplo.toml

```diff
@@ -0,0 +1,5 @@
+include = ["pyproject.toml"]
```

Comment:
For me it is unclear what the use is of this file, maybe a comment that explains?